### PR TITLE
[PPP-3577] - Removing Orient DB from CDF

### PR DESCRIPTION
**Pull request by pamval**

### CODE REVIEW

**Comment by pamval** [comment]
@diogofscmariano have no clue what wingman is on about. Please review locally and merge.

**Comment by pamval** [comment]
@diogofscmariano have no clue what wingman is on about. Please review locally and merge.

File: pentaho/src/main/java/org/pentaho/cdf/context/ContextEngine.java [hunk 1]
```diff
@@ -58,7 +52,7 @@
 
 public class ContextEngine {
 
-  static final String SESSION_PRINCIPAL = "SECURITY_PRINCIPAL";
+//  static final String SESSION_PRINCIPAL = "SECURITY_PRINCIPAL";
```
Comment (diogofscmariano) [inline pentaho/src/main/java/org/pentaho/cdf/context/ContextEngine.java]: Any need to keep this?

File: pentaho/src/main/java/org/pentaho/cdf/context/ContextEngine.java [hunk 2]
```diff
@@ -287,18 +281,13 @@ protected JSONObject buildContextConfig( final JSONObject contextObj, String ful
     return contextObj;
   }
 
+  //TODO: Clean view parameter as it's no longer being used
```
Comment (diogofscmariano) [inline pentaho/src/main/java/org/pentaho/cdf/context/ContextEngine.java]: Can we try to have this done?

File: pentaho/src/main/java/org/pentaho/cdf/context/ContextEngine.java [hunk 3]
```diff
@@ -307,6 +296,7 @@ public String getConfig( String path, String view, Map<String, String> parameter
     return s.toString();
   }
 
+  //TODO: Clean view parameter as it's no longer being used
```
Comment (diogofscmariano) [inline pentaho/src/main/java/org/pentaho/cdf/context/ContextEngine.java]: Can we try to have this done?

**Comment by pamval** [comment]
@diogofscmariano pr updated, review again

**Pull request closed without merging**